LL/senchas_síl_ébir.ttl

```diff
@@ -591,25 +591,25 @@
     owl:sameAs <http://example.com/LL/clanna_ébir_i_l-leith_chuind.ttl#Conrach>.
 
 <#EochoAnchend>
-    a foaf:Person;
+    a irishTitles:Rí;
```

Comment:
Just to check, does `a irishTitles:Rí` in any way link all the kings together and imply they were all kings of the same polity? Most of the "kings" in this section are kings of Ireland from Munster, but here the kings are of Leinster and Connacht.